model/task/task.go

```diff
@@ -1469,7 +1469,15 @@ func (t *Task) MarkEnd(finishTime time.Time, detail *apimodels.TaskEndDetail) er
 		"execution": t.Execution,
 		"project":   t.Project,
 		"details":   t.Details,
+		"status":    detail.Status,
```

Comment:
We're already logging this as part of "details"

Reply:
Oops yes we are :)